dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoFactory.java

```diff
@@ -44,13 +43,13 @@ private SourceCodeRepoFactory() {
 
     public static SourceCodeRepoInterface createGitHubAppRepo(String token) {
         // The gitUsername doesn't seem to matter
-        return new GitHubSourceCodeRepo("dockstore", token, "JWT");
+        return new GitHubSourceCodeRepo("JWT", token);
```

Comment:
Won't there be the same issue risk of GitHub apps clobbering each others' caches? May not be a enough of issue for our tests though.

Reply:
Not sure, but it also should not change. The JTW tokens are supposed to expire every ten minutes anyway so cachiing is a bit futile except for rapid requests for the same content

Reply:
There all the getBranches/getTags calls, but those probably don't add up to much.